### PR TITLE
[DESK-359] Install CLI in the windows installation and hide terminal

### PR DESCRIPTION
**Pull request by mauricio0000**
[DESK-359] Install CLI in the windows installation and hide terminal 

### Description

[DESK-359] Install CLI in the windows installation and hide terminal 

### Checklist

- [ ] Pull Request title is in the format `[PROJ-12345] Some useful description here...`
- [ ] I have prefixed this PR with `[WIP]` if it is a Work In Progress (not ready to merge)
- [ ] The git branch is in the format `PROJ-12345-some-useful-description-here`
- [ ] I have added one or more reviewers
- [ ] I have added/updated tests for any code changes
- [ ] I have updated documentation (including README, inline comments and docs.remote.it docs)
- [ ] This PR only contains one isolated change (bug fix, feature, etc)
- [ ] Translation strings added/updated for all user-visible text
- [ ] I have manually reviewed this PR to make sure only the files I intended to change were changed and nothing else
- [ ] I have notified reviewers on Slack by @ mentioning them in the `#desktop`channel
- [ ] I have moved the Jira ticket into `Resolved` state and made a note in the ticket with a link to this PR

### Test plan

1. Install in windows environtment.
2. you should see the CLI installed in the same process

### Ticket

https://remoteit.atlassian.net/browse/DESK-359

[DESK-359]: https://remoteit.atlassian.net/browse/DESK-359
[DESK-359]: https://remoteit.atlassian.net/browse/DESK-359

### CODE REVIEW

File: uninstaller.nsh [hunk 1]
```diff
@@ -1,16 +1,32 @@
+!define CLI_VERSION "1.5.3"
+!define PATH_REMOTE_CLI "C:\Program Files\remoteit-bin"
```
Comment (JamieRuderman) [inline uninstaller.nsh:2]:
```suggestion
!define REMOTE_CLI_DIR "C:\Program Files\remoteit-bin"
```

File: uninstaller.nsh [hunk 2]
```diff
@@ -1,16 +1,32 @@
+!define CLI_VERSION "1.5.3"
```
Comment (JamieRuderman) [inline uninstaller.nsh:1]: Is there a way we can put the version number in the package.json and read it here?

Comment (mauricio0000) [reply uninstaller.nsh]: @JamieRuderman it's not posible

Comment (JamieRuderman) [reply uninstaller.nsh]: Okay just thought I'd check!  :)

**Pull request merged**